docs/development/custom-ca-citadel/README.md

```diff
@@ -0,0 +1,104 @@
+# Custom CA *(intermediate one)* to be used by Citadel
+
+## Background
+
+Imagine to work for a big company owning its own PKI. Most of this companies owns its own Root CA generating
+intermediate CAs to sign Clients and/or Server Certificates.
+
+As Istio provides a way to issue certificates *(by default, self-signed)* for every workload inside a mesh,
+it has to be a way to provide a custom CA to issue workload certificates instead of using a *(default)* new one.
+
+
+## Requirements
+
+Ensure to have the following files:
+
+- `certs/ca-cert.pem`: Certificate of the Intermediate CA.
+- `certs/ca-key.pem`: Private key of the Intermediate CA.
+- `certs/root-cert.pem`: Certificate of the Root CA.
+- `certs/cert-chain.pem`: Certificate Chain. `cat certs/ca-cert.pem certs/root-cert.pem > certs/cert-chain.pem`.
+
+Also, it's required to know what `katalog/istio/sidecar-injection/configuration` it configured because it
+needs to be modified.
```

Comment:
I'm not sure I understand this sentence. Did you mean something like this?:
Additionaly, knowing how to configure `katalog/istio/sidecar-injection/configuration` is a plus, because some tunning will be needed.

Reply:
Better this way?

> Also, it's required to know what `katalog/istio/sidecar-injection/configuration` was used because some tunning will be needed.

Reply:
Yep! That's better :)